modules/highgui/CMakeLists.txt

```diff
@@ -78,6 +79,12 @@ if(HAVE_QT)
     set(qt_deps Core Gui Widgets Test Concurrent)
     if(HAVE_QT_OPENGL)
       add_definitions(-DHAVE_QT_OPENGL)
+      # QOpenGLWidget requires Qt6 package component OpenGLWidgets
+      if(QT_VERSION_MAJOR GREATER 5)
+        # Qt6OpenGLWidgets_DEFINITIONS,Qt6OpenGLWidgets_INCLUDE_DIRS, and
+        # Qt6OpenGLWidgets_INCLUDE_LIBRARIES not specified in Qt 6.2.1 build
+        list(APPEND HIGHGUI_LIBRARIES Qt6::OpenGLWidgets)
```

Comment:
Why `list(APPEND qt_deps OpenGLWidgets)` doesn't work?

Reply:
I printed the Qt6OpenGLWidgets_DEFINITIONS, Qt6OpenGLWidgets_INCLUDE_DIRS, and Qt6OpenGLWidgets_INCLUDE_LIBRARIES variables during CMake configuration and they were empty. A little bit puzzled as to why since I see them in the Qt config files, e.g.

./lib/cmake/Qt6OpenGLWidgets/Qt6OpenGLWidgetsConfig.cmake:98:             ${Qt6${_module_dep}_INCLUDE_DIRS})

I will look further into this, any suggestions would be appreciated.

Reply:
Interesting, my mac build is finding these variables: Qt6OpenGLWidgets_DEFINITIONS, Qt6OpenGLWidgets_INCLUDE_DIRS, and Qt6OpenGLWidgets_INCLUDE_LIBRARIES. 

Maybe had a typo. In any case, I'm going to push up the edits and then test build on Windows. Local high-gui test on the mac runs fine.